pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -3158,11 +3190,21 @@ def set_fullband(self, band):
         'set_full_span' command is activated, the span is limited to the frequency band of the
         selected harmonic.
         """,
-        validator=joined_validators(strict_range, strict_discrete_set),
-        values=[[int(1), int(54)], ["ON", "OFF"]],
+        validator=strict_range,
+        values=[int(1), int(54)],
```

Comment:
`1`  is already an integer, equally `54`, so `int(1)` is redundant.

I understand, that you are touching the validators, not the values. Feel free to refactor the code or not.